apps/server/src/modules/lesson/index.ts

```diff
@@ -1,5 +1,4 @@
 export * from './lesson.module';
-export * from './service/lesson-copy.service';
-export * from './service/lesson.service';
 export * from './types/lesson-copy-parent.params';
 export * from './types/lesson-copy.params';
+export * from './service';
```

Comment:
The switch to ./service is well, but then refactor it one step more and also change the wildcard to specific output like in learnroom/index.ts

Reply:
@WojciechGrancow

Reply:
done